includes/apple-exporter/components/class-facebook.php

```diff
@@ -85,6 +96,21 @@ public static function node_matches( $node ) {
 			}
 		}
 
+		// Handling for a rendered WordPress.com Facebook embed.
+		$fb_post = $node->getElementsByTagName( 'post' );
+		if ( ! empty( $fb_post->length ) && 1 === $fb_post->length ) {
+
+			// Extract Facebook URL from element's href property.
+			$fb_url = $fb_post[0]->getAttribute( 'href' );
+
+			// Ensure we have a valid Facebook embed url.
+			if ( ! empty( $fb_url )
```

Comment:
personal pref, you could push `! empty( $fb_url )` to it's own line.

Reply:
Yeah, that's how we do it in JS-land, but I follow the OG PEAR standards here since WP is a superset of PEAR and doesn't have specific guidance for this case: http://pear.php.net/manual/en/standards.control.php#standards.control.splitlongstatements